client/lib/products-values/constants.js

```diff
@@ -93,6 +105,8 @@ export const getJetpackProductsShortNames = () => {
 		[ PRODUCT_JETPACK_SEARCH_MONTHLY ]: translate( 'Search' ),
 		[ PRODUCT_JETPACK_SCAN ]: translate( 'Daily Scan' ),
 		[ PRODUCT_JETPACK_SCAN_MONTHLY ]: translate( 'Daily Scan' ),
+		[ PRODUCT_JETPACK_ANTI_SPAM ]: translate( 'Anti-Spam' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **41** times:
`translate( 'Anti-spam' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-anti-spam-plans-page)